docker-compose.demo.yml

```diff
@@ -20,6 +22,20 @@ services:
       - demo.env
     networks:
       - extbridge
+  gin:
+    depends_on: 
```

Comment:
I added `depends_on` so that the startup command would be `docker-compose -f ... up gin`, instead of `... up gin svc`. I don't really see a negative in having it, but I'm not a docker-compose expert so please fill me in if there are downsides I'm missing.